files/default/sql/ddl/3.1.0__initial_tables.sql

```diff
@@ -1177,8 +1177,8 @@ CREATE TABLE `serving` (
                            `local_dir` varchar(255) COLLATE latin1_general_cs DEFAULT NULL,
                            `batching_configuration` varchar(255) COLLATE latin1_general_cs DEFAULT NULL,
                            `optimized` tinyint(4) NOT NULL DEFAULT '0',
-                           `instances` int(11) NOT NULL DEFAULT '0',
-                           `transformer_instances` int(11) DEFAULT NULL,
+                           `instances` FLOAT NOT NULL,
```

Comment:
Why are we changing instances and transformer_instances to FLOAT? In hopsworks-ee PR they are still of Integer type and that makes more sense since why would an instances field have a the fractional number?

Reply:
You are right, I forgot to revert those changes. Fixed.